app/src/main/kotlin/com/wire/android/migration/preference/ScalaServerConfigDAO.kt

```diff
@@ -29,7 +29,7 @@ class ScalaServerConfigDAO @Inject constructor(private val scalaBackendPreferenc
         @Suppress("ComplexCondition")
         return if (!apiBaseUrl.isNullOrEmpty() && !accountsBaseUrl.isNullOrEmpty() && !webSocketBaseUrl.isNullOrEmpty() &&
             !blackListUrl.isNullOrEmpty() && !teamsUrl.isNullOrEmpty() && !websiteUrl.isNullOrEmpty() && !title.isNullOrEmpty()
-        ) ServerConfig.Links(apiBaseUrl, accountsBaseUrl, webSocketBaseUrl, blackListUrl, teamsUrl, websiteUrl, title, false)
+        ) ServerConfig.Links(apiBaseUrl, accountsBaseUrl, webSocketBaseUrl, blackListUrl, teamsUrl, websiteUrl, title, false, null)
```

Comment:
I suggest for this one also have a default value.

Reply:
I think I will leave this here, as I guess it will be updated with the backup things , need to check this with @saleniuk after merge